conf/standalone.conf

```diff
@@ -180,6 +180,8 @@ maxTopicsPerNamespace=0
 # 'is_allow_auto_update_schema' of namespace policy.
 isAllowAutoUpdateSchemaEnabled=true
 
+schemaRegistryClass=
```

Comment:
It's better to have a description, by default pulsar will use the internal schema registry which is based on the bookkeeper.

Reply:
@codelipenghui i have added it now